lms/djangoapps/certificates/views/webview.py

```diff
@@ -101,7 +101,12 @@ def _update_certificate_context(context, user_certificate, platform_name):
     # Translators:  The format of the date includes the full name of the month
     course = get_course_by_id(user_certificate.course_id) if user_certificate.course_id else None
 
-    if course and not course.self_paced and course.certificate_available_date:
+    if all([
```

Comment:
I think you can't use `all()` here either - you want the expression evaluation to halt if course is Falsey.

(Now I feel kind of bad for bringing up the `all`/`any` functions.  We (programmers) tend to rely on that early evaluation halting more than I'd like to think.)

Reply:
@iloveagent57 You're right. Dammit, I'll find a place to shoehorn it in!